deepset_cloud_sdk/workflows/async_client/async_pipeline_client.py

```diff
@@ -1,6 +1,4 @@
-"""Pipeline client for importing pipelines and indexes to deepset AI Platform."""
-import asyncio
-
+"""Async pipeline client for importing pipelines and indexes to deepset AI Platform."""
```

Comment:
nit. If you want you can call this async_client.pipeline_client (so no need to add an other async)